indico/modules/events/papers/models/revisions.py

```diff
@@ -182,3 +182,22 @@ def get_reviewed_for_groups(self, user, include_reviewed=False):
         if self.paper.cfp.layout_reviewing_enabled and user in self.paper.cfp.layout_reviewers:
             reviewed_for.add(PaperReviewType.layout)
         return set(map(PaperTypeProxy, reviewed_for))
+
+    def has_user_reviewed(self, user, review_type=None):
```

Comment:
Oh I see, you call it from a template... You could go for something like this:

    review_type = PaperReviewType[review_type] if isinstance(review_type, basestring) else review_type